sched/task/task_start.c

```diff
@@ -66,17 +66,18 @@
 void nxtask_start(void)
 {
   FAR struct tcb_s *tcb = this_task();
+  const uint8_t ttype = tcb->flags & TCB_FLAG_TTYPE_MASK;
```

Comment:
remove const

Reply:
it is to make `ttype` read-only in the rest of the function, why remove it?

Reply:
normally, other pointer add const